app/ts/components/Transfer.tsx

```diff
@@ -1,177 +1,194 @@
 import { batch, Signal, useComputed, useSignal, useSignalEffect } from '@preact/signals'
 import { getAddress } from 'ethers';
-import { JSX } from 'preact/jsx-runtime';
 import { connectBrowserProvider, ProviderStore } from '../library/provider.js';
-import { itentifyAddress } from '../library/identifyTokens.js'
+import { itentifyAddress, ERC721, ERC1155 } from '../library/identifyTokens.js'
 import { BlockInfo } from '../library/types.js';
 import { Button } from './Button.js';
-import { TokenPicker } from './TokenPicker.js';
-import { transferNft } from '../library/transactions.js';
-import Blockie from './Blockie.js';
+import { transferERC1155, transferERC721 } from '../library/transactions.js';
+
 import { knownNetworks } from '../library/networks.js'
-import { BlockieTextInput, NumberInput, TextInput } from './Inputs.js';
+import { BlockieTextInput, NumberInput, TextInput, TokenAmountInput } from './Inputs.js';
+import { ItemDetails } from './ItemDetails.js';
 
 export const Transfer = ({ provider, blockInfo }: { provider: Signal<ProviderStore | undefined>, blockInfo: Signal<BlockInfo> }) => {
-	const showTokenPicker = useSignal<boolean>(false)
-	const selectedNft = useSignal<{ address: string, id: bigint, owner: string, name?: string, symbol?: string, tokenURI?: string } | undefined>(undefined)
+	// const showTokenPicker = useSignal<boolean>(false)
+	const selectedNft = useSignal<ERC721 | ERC1155 | undefined>(undefined)
+
+	// 'notfound' | 'badid' | 'noprovider' | 'EOA' | 'contract' | 'ERC20' | 'opensea'
```

Comment:
is the plan to add support for these, or are these old?

Reply:
Need to clean this up in #15